stat_fs_test.go

```diff
@@ -0,0 +1,33 @@
+package hdfs
+
+import (
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestStatFs(t *testing.T) {
+	t.Skip("Stat FS does not work on Travis CI")
```

Comment:
But for now, it's probably fine to just remove the shelling out and make sure the command succeeds at all.